src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
```

Comment:
Avoid using static methods as per the checklist. Instead, you should create an instance of the class to call its methods. Static methods are generally considered bad practice in object-oriented programming because they can lead to less flexible and more tightly coupled code.